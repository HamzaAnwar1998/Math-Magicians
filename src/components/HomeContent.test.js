import renderer from 'react-test-renderer';

it('renders HomeContent heading', () => {
  const tree = renderer.create(<h1>Welcome to our page!</h1>).toJSON();
  expect(tree).toMatchSnapshot();
});
