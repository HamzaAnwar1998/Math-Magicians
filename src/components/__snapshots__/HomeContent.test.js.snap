// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HomeContent heading 1`] = `
<h1>
  Welcome to our page!
</h1>
`;
